notebooks/chapter03/section03_tokenizer.py

```
@@ -0,0 +1,408 @@
+    truncated_tokens = truncate_tokens(long_tokens, max_length)
+    print("Truncated tokens:", truncated_tokens)
+
+
+def demonstrate_embeddings():
+    """
+    エンベディングのデモンストレーション
+    """
+    print("\n=== エンベディングのデモ ===")
+
+    # 入力エンベディング
+    embedding_layer, input_embeddings = create_input_embedding_example()
```

Ruff (0.16.1)

[warning] Unpacked variable `embedding_layer` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)